ts2mls/js/src/main/scala/ts2mls/JSWriter.scala

```diff
@@ -0,0 +1,40 @@
+package ts2mls
+
+import scala.scalajs.js
+import js.Dynamic.{global => g}
+import js.DynamicImplicits._
+
+class JSWriter(filename: String) {
+  import JSWriter._
+  private val out = fs.openSync(filename, "rs+")
+  private var fileSize = 0 // how many bytes we've written in the file
+  private var needTruncate = false
+
+  def writeln(str: String) = {
+    val strln = str + "\n"
+    val buffer = createBuffer(strln.length)
+    fs.readSync(out, buffer, 0, strln.length)
+    
+    // override when the content is different
+    if (!strln.equals(buffer.toString())) {
```

Comment:
Style: use `===` after importing `mlscript.utils._` instead.

Reply:
I've replaced all `something.equals` and `!something.equals` with `===` and `=/=`. Is it necessary to use abbreviation defined in `shorthands.scala` either?

Reply:
No it's not necessary, but you can do it at your own convenience.

Reply:
I still prefer full names. So I guess I can keep them. 😄